common/models.py

```diff
@@ -38,3 +38,30 @@ class User(AbstractUser, TimestampedModel):
     
     def __str__(self):
         return self.email
+
+# START_FEATURE django_storages
+# from django.conf import settings
+# from django.core.files.storage import FileSystemStorage
+# from storages.backends.s3boto3 import S3Boto3Storage
+# 
+# def get_file_storage():
```

Comment:
This function should not be necessary.  Unless you are overriding the default file storage, you don't need to pass this argument.  The place to define FileSystemStorage is in settings.py